ui/index.d.ts

```diff
@@ -630,3 +630,8 @@ export interface UIExtensionVariables {
     ui: IAzureUserInput;
     reporter: TelemetryReporter | undefined;
 }
+
+/**
+ * Adds a user agent specific to the VS Code extension, of the form `${extensionName}/${extensionVersion}`
+ */
+export declare function addExtensionUserAgentInfo(client: ServiceClient): void;
```

Comment:
nit: I'd say the "Info" part is unnecessary

Reply:
Fine with me. I was just being consistent with `client.addUserAgentInfo`